### PR TITLE
Add manifests for Tailscale

### PR DESCRIPTION
**Pull request by agottardo**
This PR adds manifests and icons for both the Mac App Store and Standalone variants of Tailscale for macOS.

### CODE REVIEW

**Review by apizz — CHANGES_REQUESTED** [review]

File: Manifests/ManagedPreferencesApplications/io.tailscale.ipn.macos.plist [hunk 1]
```diff
@@ -0,0 +1,434 @@
...
+<dict>
+	<key>pfm_app_url</key>
+	<string>https://apps.apple.com/app/tailscale/id1475387142</string>
+	<key>pfm_documentation_url</key>
+	<string>https://tailscale.com/kb/1315/mdm-keys/</string>
+	<key>pfm_targets</key>
+	<array>
+		<string>system</string>
+		<string>user</string>
+	</array>
```
[the remark this answers is not in the document]
Comment (agottardo) [reply Manifests/ManagedPreferencesApplications/io.tailscale.ipn.macos.plist]: done!

File: Manifests/ManagedPreferencesApplications/io.tailscale.ipn.macos.plist [hunk 2]
```diff
@@ -0,0 +1,434 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>pfm_app_url</key>
+	<string>https://apps.apple.com/app/tailscale/id1475387142</string>
+	<key>pfm_description</key>
+	<string>Configure settings for the Mac App Store variant of the Tailscale client.</string>
```
Comment (apizz) [inline Manifests/ManagedPreferencesApplications/io.tailscale.ipn.macos.plist:8]: Is this really only for the App store version and not also the PKG version of Tailscale?

**Review by apizz — COMMENTED** [review]
I think the only ?

File: Manifests/ManagedPreferencesApplications/io.tailscale.ipn.macos.plist (hunk 2 above)
Comment (apizz) [reply Manifests/ManagedPreferencesApplications/io.tailscale.ipn.macos.plist:8]: disregard, I forgot different domains

**Review by apizz — APPROVED** [review]
LGTM! Thanks for contributing ❤️  love when devs contribute to these things :)

**Pull request merged**